client/src/components/Container.jsx

```diff
@@ -23,10 +24,10 @@ class Container extends React.Component {
     return (
       <div style={styles.container}>
         <h2 style={styles.header}>Reviews</h2>
-        <Review />
-        <Review />
-        <Review />
-        <Review />
+        <ReviewContainer />
```

Comment:
Why do you need to render it 4 times?

Reply:
This will change once I stop hardcoding it. This is just so that there are 4 reviews showing like there are on etsy.